Cargo.toml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+resolver = "2"
+members = [
+    "up-streamer",
+]
+
+[workspace.package]
+rust-version = "1.66.1"
+version = "0.1.5-dev" # uProtocol version
+repository = "https://github.com/eclipse-uprotocol/up-streamer-rust"
+homepage = "https://github.com/eclipse-uprotocol"
+authors = [
+    "Pete LeVasseur <plevasseur@gmail.com>",
+]
+edition = "2021"
+keywords = ["uProtocol", "SDV", "routing", "streamer"]
+license = "Apache-2.0"
+
+[workspace.dependencies]
+async-std = { version = "1.12.0", features = ["unstable", "attributes"] } # unstable feature needed for task::spawn_local()
+async-trait = { version = "0.1" }
+env_logger = { version = "0.10.1" }
+futures = { version = "0.3.30" }
+log = { version = "0.4.20" }
+prost = { version = "0.12" }
+prost-types = { version = "0.12" }
+serde_json = { version = "1.0.111" }
+uuid = { version = "1.7.0" }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "c705ac97602ad6917a93d23651e8a504ec7bb718" }
```

Comment:
Do you need/want all the default features? If not, you might want to disable default features here and turn on only those in the workspace members that are really needed ...

Reply:
Good point! I think we have them all turned off for now by default, but best to pick and choose as needed.